client/datasync-client/example/index.ts

```diff
@@ -42,12 +42,12 @@ $fh.sync.notify(datasetId, function(notification) {
   if ('sync_complete' === code) {
     $fh.sync.doList(datasetId,
       function(res) {
-        // tslint:disable-next-line:no-console
-        console.log('Successful result from list:', JSON.stringify(res));
+        logger.info('Successful result from list:', JSON.stringify(res),
+          {level: 'INFO', tag: 'client:datasync-client:example', src: 'index.ts'});
       },
       function(err) {
-        // tslint:disable-next-line:no-console
-        console.log('Error result from list:', JSON.stringify(err));
+        logger.error('Error result from list:', JSON.stringify(err),
```

Comment:
Why we do ` JSON.stringify` that's antipattern. You should be able to pass object and it hsould be stringified